src/components/AccessibilityPolicyForm/AccessibilityPolicyForm.test.jsx

```diff
@@ -157,7 +156,7 @@ describe('<AccessibilityPolicyForm />', () => {
       expect(wrapper.state('isStatusAlertOpen')).toEqual(true);
       expect(statusAlertType).toEqual('success');
       expect(statusAlert.find('div').first().hasClass('alert-success')).toEqual(true);
-      expect(statusAlert.text()).toContain('Thank you for contacting edX!Thank you for your feedback regarding the accessibility of Studio. We typically respond within one business day (Monday to Friday, 13:00 to 21:00 UTC).');
+      expect(statusAlert.find('div').first().text()).toContain('×Thank you for contacting edX!Thank you for your feedback regarding the accessibility of Studio. We typically respond within one business day (Monday to Friday, 1:00 PM UTC to 9:00 PM UTC).');
```

Comment:
What is the funny looking x for? Is it the x button of the StatusAlert? It seems kind of odd that it comes through in the call to text().

Reply:
It's this `&times;` entity: https://github.com/edx/paragon/blob/b783247b0133fc5c64d2d58561c08541cb3bf0b3/src/StatusAlert/index.jsx#L75

`statusAlert` is this whole div, so the `x` icon is included in the return value of text(): 
<img width="945" alt="screen shot 2018-02-05 at 12 22 34 pm" src="https://user-images.githubusercontent.com/7373924/35819087-ad99a564-0a6f-11e8-810b-c5eb5c396200.png">

I figured that it would make for a more complete test to include this value in the check, we can easily remove if if we want though since this is a `toContain` check

Reply:
Ah, I see. That makes sense. I just didn't realize that the div would include the x in the button. I thought it was just testing the alert-dialog.